integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/distrox/DistroXEncryptedVolumeTest.java

```diff
@@ -366,6 +380,18 @@ private void verifyVolumeEncryptionKey(CloudPlatform cloudPlatform, String resou
                     throw new TestFailException(format("Volume has not been encrypted with '%s' key!", desKeyUrl));
                 }
             });
+        } else if (CloudPlatform.GCP.equals(cloudPlatform)) {
+            String kmsKeyArn = gcpCloudProvider.getEncryptionKey(true);
```

Comment:
Here you can apply a simply solution instead:
```
            String kmsKeyName = gcpCloudProvider.getEncryptionKey(true);
            List<String> volumeKmsKeyIds = new ArrayList<>(cloudFunctionality.listVolumeEncryptionKeyIds(resourceName, null, instanceIds));
            if (volumeKmsKeyIds.stream().noneMatch(keyId -> StringUtils.containsIgnoreCase(keyId, kmsKeyName))) {
                LOGGER.error(format("Volume has not been encrypted with '%s' KMS key!", kmsKeyName));
                throw new TestFailException(format("Volume has not been encrypted with '%s' KMS key!", kmsKeyName));
            } else {
                LOGGER.info(format("Volume has been encrypted with '%s' KMS key.", kmsKeyName));
                Log.then(LOGGER, format(" Volume has been encrypted with '%s' KMS key. ", kmsKeyName));
            }
```
and almost the same is true for AZURE, as a refactor:
```
            List<String> volumesDesIds = new ArrayList<>(cloudFunctionality.listVolumeEncryptionKeyIds(resourceName, resourceGroupName, instanceIds));
            if (volumesDesIds.stream().noneMatch(desId -> StringUtils.containsIgnoreCase(desId, "diskEncryptionSets/" + environmentName))) {
                LOGGER.error(format("Volume has not been encrypted with '%s' key!", desKeyUrl));
                throw new TestFailException(format("Volume has not been encrypted with '%s' key!", desKeyUrl));
            } else {
                LOGGER.info(format("Volume has been encrypted with '%s' DES key.", desKeyUrl));
                Log.then(LOGGER, format(" Volume has been encrypted with '%s' DES key. ", desKeyUrl));
            }
```